apps/admin/src/app/app.spec.tsx

```diff
@@ -10,6 +10,6 @@ describe('App', () => {
 
   it('should have a greeting as the title', () => {
     const { getByText } = render(<App />);
-    expect(getByText(/Welcome admin/gi)).toBeTruthy();
+    expect(getByText(/Welcome to our admin app./gi)).toBeTruthy();
```

Comment:
Here you can see how is `app.tsx` chages, also is needed to update spec.tsx file in order to pass tests if its a worth change.